packages/create-neutrino-project/commands/init/templates/eslintrc.js

```diff
@@ -0,0 +1,3 @@
+const { Neutrino } = require('neutrino');
+
+module.exports = Neutrino().call('eslintrc');
```

Comment:
Could you update the docs to use this shorter `.eslintrc.js` template too?

Reply:
Addressed in https://github.com/mozilla-neutrino/neutrino-dev/pull/412 :)